docs/_config.yml

```diff
@@ -5,15 +5,15 @@ repository: /SAP/fundamental
 output: web
 
 # this appears on the top navigation bar next to the home button
-topnav_title: Fiori Fundamentals
+topnav_title: SAP Fundamentals
 
 # this appears in the html browser tab for the site title (seen mostly by search engines, not users)
-site_title: SAP Fiori Fundamentals
+site_title: SAP Fundamentals
 
-site_description: SAP Fiori Fundamentals is a Fiori 3 component library and SASS toolkit for building SAP user interfaces with any technology.
+site_description: SAP Fundamentals is a HTML/CSS component library and SASS toolkit for building SAP user interfaces with any technology.
```

Comment:
I think we can keep here the statement that it allows you to create Fiori 3 components